client/src/app/core/actions/has-create-update-delete-action.ts

```diff
@@ -0,0 +1,9 @@
+import { Identifiable } from 'app/shared/models/base/identifiable';
+import { BaseViewModel } from 'app/site/base/base-view-model';
+import { Id } from '../definitions/key-types';
+
+export interface CreateUpdateAction<V extends BaseViewModel> {
```

Comment:
Not used, can be deleted.

Again: create with a `Partial<V>` is dangerous, since we often have extra non-model properties to send to the server. This will not work.